app/controllers/entries_controller.rb

```diff
@@ -2,8 +2,11 @@ class EntriesController < ApplicationController
   before_action :authenticate
 
   def index
-    if current_week?
-      @entries = Entry.in_current_week.stopped.includes(:project, :user)
+    if current_week? || current_month?
+      @entries = Entry.stopped.includes(:project, :user)
+      @entries = @entries.in_current_month if current_month?
+      @entries = @entries.in_current_week if current_week?
+      @entries = @entries.where(user_id: filter_user_id) if filter_user_id
```

Comment:
I prefer to use private methods like `find_entries_from_current_week_or_month` to prevent heavy action methods, but OK